apps/prairielearn/src/lib/server-jobs.ts

```diff
@@ -33,20 +33,24 @@ interface ServerJobExecOptions {
   env?: NodeJS.ProcessEnv;
 }
 
+export interface ServerJobResult {
+  data: Record<string, any>;
+}
+
 export interface ServerJob {
   fail(msg: string): never;
   error(msg: string): void;
   warn(msg: string): void;
   info(msg: string): void;
   verbose(msg: string): void;
-  exec(file: string, args?: string[], options?: ServerJobExecOptions): Promise<void>;
+  exec(file: string, args?: string[], options?: ServerJobExecOptions): Promise<ServerJobResult>;
```

Comment:
I'd like @nwalters512's eyes on this as a change to the core server jobs code.

Reply:
Discussed this afternoon.